views/js/runner/plugins/navigation/limitBackButton.js

```diff
@@ -111,12 +111,6 @@ define([
                     toggleBackButton(disableState);
                 });
 
-                testRunner.after('disablenav.' + pluginName, function () {
-                    if (disableState) {
-                        testRunner.trigger('enablenav');
```

Comment:
I think the initial goal was to enable back the navigation when the user tried to click on a disabled previous. 
This line should be copied just before the rejection : 
https://github.com/oat-sa/extension-tao-test-runner-plugins/pull/124/files#diff-1384560f4a2822885af8c5d321adc29aL106